app/views/variable_lookup_keys/edit.html.erb

```diff
@@ -1,2 +1,17 @@
 <% title(_("Edit Smart Variable")) %>
+<%= breadcrumbs(
+    items:
+    [
+      {
+        caption: _("Smart variables"),
+        url: (url_for(variable_lookup_keys_path) if authorized_for(hash_for_variable_lookup_keys_path))
+      },
+      {
+        caption: _('Edit Smart Variable')
+      }
+    ],
+    resource_url: "/api/v2/smart_variables/",
+    switchable: true,
+    switcher_item_url: "/variable_lookup_keys/:id/edit"
```

Comment:
I'm not sure if you can use an helper for the switcher_item_url, as the id is later on being replaced by javascript code.. I would think the url helpers will fail without an AR object?

Reply:
You can do it, see my first comment: https://github.com/theforeman/foreman/pull/5396#discussion_r180413214
The js code replaces `:id` with the actual value. When you pass `":id"`to the helper instead of the AR object, it generates `"/variable_lookup_keys/:id/edit"` (I tested it).

Reply:
^ we will create a smaller PRs later in order to fix it.